static/app/views/pipeline/coverage/coverageTrend.tsx

```diff
@@ -0,0 +1,16 @@
+import styled from '@emotion/styled';
+
+import {space} from 'sentry/styles/space';
+
+export default function CoverageTrendPage() {
+  return (
+    <LayoutGap>
+      <p>Coverage Trend</p>
```

Comment:
Shouldn't this be `t('Coverage Trend')`? Not sure if since this is placeholder text

Reply:
Yea this is just a place holder for us internally.